### PR TITLE
add mentors to Numerics projects

### PR DESCRIPTION
**Pull request by shashi**
ping @jiahao @simonbyrne @andreasnoack @jaredcrean2

### CODE REVIEW

File: soc/projects/numerics.md [hunk 1]
```diff
@@ -82,6 +93,8 @@ and boundary cases.
 
 **Expected Results**: New and faster methods for evaluating properties of special functions.
 
+**Mentors:** [Alex Arslan](https://github.com/ararslan)
```
Comment (ararslan) [inline soc/projects/numerics.md]: Oh hey, that's me. What does being a mentor entail? Is there some guiding documentation I can consult?

Comment (ararslan) [inline soc/projects/numerics.md]: Also I'm probably not the best person to consult in regards to developing the special functions code. That'd probably be @simonbyrne, if he's able.

Comment (simonbyrne) [inline soc/projects/numerics.md]: I'm happy to do it.

**Comment by andreasnoack** [comment]
We might want @Sacha0 if he would be interested in mentoring a sparse project.

**Comment by Sacha0** [comment]
> We might want to ask `@Sacha0` if he would be interested in mentoring a sparse project.

Bandwidth allowing, I'd be delighted. Best!

**Comment by JaredCrean2** [comment]
Same situation as ararslan, need more info

File: soc/projects/numerics.md (hunk 1 above)
Comment (shashi) [inline soc/projects/numerics.md]: Great! @ararslan there is no guiding documentation about this wrt to Julia yet but there is http://archive.flossmanuals.net/GSoCMentoring/what-makes-a-good-mentor and other pages in that manual which explain some responsibilities. I believe mentors get about $500 for their help.

**Comment by shashi** [comment]
Thanks @Sacha0 ! I've updated the mentors list. @JaredCrean2 See my reply to Alex above and the link in it :) I'm merging this now, let me know if you want your name removed, I'll do it in a later commit.

File: soc/projects/numerics.md (hunk 1 above)
Comment (ararslan) [inline soc/projects/numerics.md]: Thanks for the link! Maybe next year I'll volunteer to mentor for a project I'm more intimately familiar with. 🙂

**Comment by JaredCrean2** [comment]
This looks like a bigger time commitment than I can handle this summer, so I must bow out.

**Comment by shashi** [comment]
Removed. Thanks!

**Pull request merged**